src/tasks/spatial_decomposition/methods/destvi/config.vsh.yaml

```diff
@@ -33,8 +33,7 @@ platforms:
     setup:
```

Comment:
change image for this task to `ghcr.io/openproblems-bio/base_pytorch_nvidia:1.0.3`